android/src/main/java/com/onegini/mobile/sdk/reactnative/RNOneginiSdk.kt

```diff
@@ -213,7 +217,8 @@ class RNOneginiSdk(reactContext: ReactApplicationContext) : ReactContextBaseJava
     }
 
     @ReactMethod
-    override fun submitCustomRegistrationAction(customAction: String, identityProviderId: String, token: String?) {
+    fun submitCustomRegistrationAction(customAction: String, identityProviderId: String, token: String?) {
+        //TODO: Add the nullchecks here and make params nullable
```

Comment:
I guess you missed this one when adding nullchecks

Reply:
This function is not promise based yet, so we can't really add the nullchecks here.